### PR TITLE
cleaned convertpdf

### PR DESCRIPTION
**Pull request by Amet015**

### CODE REVIEW

File: src/main/java/com/jalasoft/jfc/model/pdf/PdfConverter.java [hunk 1]
```diff
@@ -70,30 +70,22 @@
     public FileResponse convert(Param param) throws CommandValueException, ConvertException, ZipJfcException, IOException {
```
Comment (JuanCarlosMartinez4) [inline src/main/java/com/jalasoft/jfc/model/pdf/PdfConverter.java:70]: delete IOException if its possible

File: src/main/java/com/jalasoft/jfc/model/pdf/PdfConverter.java [hunk 2]
```diff
@@ -70,30 +70,22 @@
...
         }
-        if (pdfParam.isMetadata()) {
-            stringCommand = new StringBuilder();
-            stringCommand.append(generateImage(pdfParam));
-            runCommand(stringCommand.toString());
-            generateMetadata(pdfParam);
-        }
+
         if (param.isMetadata()) {
             MetadataConverter metadataConverter = new MetadataConverter();
```
Comment (JuanCarlosMartinez4) [inline src/main/java/com/jalasoft/jfc/model/pdf/PdfConverter.java:85]: Is it correct the class name "MetadataConverter" ?, actually that class gets file's metadata

Comment (Amet015) [reply src/main/java/com/jalasoft/jfc/model/pdf/PdfConverter.java:85]: yes, it is correct

File: src/main/java/com/jalasoft/jfc/model/pdf/PdfConverter.java (hunk 1 above)
Comment (Amet015) [reply src/main/java/com/jalasoft/jfc/model/pdf/PdfConverter.java:70]: it is no possible, because metadata uses it

**Review by JuanCarlosMartinez4 — APPROVED** [review]

**Pull request merged**